resources/js/Types/ButtonFrameProps.d.ts

```diff
@@ -0,0 +1,6 @@
+export interface ButtonFrameProps {
```

Comment:
Did you need `export` when this is a `d.ts` file ?

Reply:
Yes, it doesn't work without it
![Screenshot from 2024-11-04 12-18-09](https://github.com/user-attachments/assets/11e07896-df0a-4096-be22-7983b5f22b72)